### ISSUE TITLE
loss_tr vs. loss_val when there is no cross-validation

### ISSUE THREAD

**Issue by woctezuma** [MIC-DKFZ/nnUNet#114]
I have trained nnU-Net with a small training dataset of 5 images from [the CHAOS challenge](https://chaos.grand-challenge.org/Combined_Healthy_Abdominal_Organ_Segmentation/). I only wanted to try the algorithm, so I deactivated cross-validation (setting `FOLD ` to `all`). Here is the plot which is created by the program:

![Training](https://raw.githubusercontent.com/wiki/woctezuma/playing-with-simpleitk/img/training.png)

As you can notice, `loss_tr` and `loss_val` are not superimposed, even though there is no cross-validation. I wonder the reason why. Is it due to an averaging of the loss over a batch vs. over the whole dataset? What would explain the discrepancy? Is the validation dataset not equal to the training dataset in this scenario?

Moreover, I think "evaluation metric" is the average Dice score for the training dataset. Is that correct? As I mentioned in #111, it would be nice to clarify what "evaluation criterion" or "evaluation metric" is.

**Comment by FabianIsensee** [comment]
> As you can notice, loss_tr and loss_val are not superimposed, even though there is no cross-validation. I wonder the reason why. 

This is because the train loss is computed on samples that receive data augmentation and the validation loss is computed on samples that don't get data augmentation

> Moreover, I think "evaluation metric" is the average Dice score for the training dataset. Is that correct? As I mentioned in #111, it would be nice to clarify what "evaluation criterion" or "evaluation metric" is.

evaluation metric is nothing to be relied upon. It is the average 'global dice', computed for eqach class independently. What I mean by global Dice is that the FP FN and TP are accumulated over all patches in the validation and at the end of the validation the Dice score is computed from those. You should interpret this as an optimistic estimate

Best,
Fabian
